shiptendance/views/edit.ejs

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Pirate Editor</title>
+  </head>
+  <body>
```

Comment:
Next time try to use partials so you can clean up your code and make it DRY without repeating yourself in every view.